packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -34,6 +34,17 @@ const PipelineRuns = ({
   createPipelineRunTimestamp = pipelineRun =>
     getStatus(pipelineRun).lastTransitionTime,
   createPipelineRunsByPipelineURL = urls.pipelineRuns.byPipeline,
+  getPipelineRunStatus = (pipelineRun, intl) =>
```

Comment:
Having 'PipelineRun' in all of the prop names on a component called PipelineRuns seems redundant, I get why we have it for the URLs to differentiate between the 2 cases, but for the rest it seems a bit much.

What do you think about renaming these to `getStatus` and `getStatusIcon`?  We'd just have to rename the imports above and make sure existing references to those functions are updated in this component.

Reply:
Discussed offline, we'll leave it as-is for now and revisit if it looks like an issue when preparing our first official release of the component library.